bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -173,7 +194,7 @@ if wflow.cp.has_section("workflow-injections"):
     # Generate injection matched filter workflow
     inj_insp_files = _workflow.setup_matchedfltr_workflow(wflow, sciSegs,
             datafind_veto_files, inj_splitbank_files, injDir, injs,
-            tags=["injections"])
+            tags=[mf_tag + "_injections"])
```

Comment:
I think these should be two tags. Ie 
```
tags=[mf_tag,"injections"]
```

This then let's you specify options specific for each tag (ie. the injection-specfic options don't care on the mf_tag and vice-versa).